.github/workflows/publish.yaml

```diff
@@ -0,0 +1,55 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      next_version:
+        description: 'version (e.g. 3.4.0)'
+        required: true
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Configure Git Credentials
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
Let's use Node 18 as 16 is no longer maintained

Reply:
I updated the version to 18, but I believe we are still recommending to use version 16 in the readme